src/Compiler/Driver/CompilerOptions.fsi

```diff
@@ -52,7 +52,16 @@ val FilterCompilerOptionBlock: (CompilerOption -> bool) -> CompilerOptionBlock -
 /// Parse and process a set of compiler options
 val ParseCompilerOptions: (string -> unit) * CompilerOptionBlock list * string list -> unit
 
-val DisplayBannerText: TcConfigBuilder -> unit
+val DisplayBannerText: tcConfigB: TcConfigBuilder -> printer: (string -> unit) -> unit
+
+val DisplayHelpFsc:
```

Comment:
I think entry points revealed only for unit testing should be systematically named, e.g. `DisplayHelpFsc_Testable` or something.